extensions/geode-modules/build.gradle

```diff
@@ -45,7 +45,9 @@ dependencies {
   testImplementation('org.assertj:assertj-core')
   testImplementation('org.mockito:mockito-core')
   testImplementation('org.apache.tomcat:catalina-ha:' + DependencyConstraints.get('tomcat6.version'))
-
+  testImplementation(project(':geode-junit')) {
```

Comment:
I don't see what symbol from `geode-junit` is being used to require this addition.